frontend/src/routes/api/formation/quiz/createQuiz/+server.ts

```
@@ -0,0 +1,22 @@
+import FormationService from '$lib/services/FormationService';
+
+// API endpoint : /api/formation/quiz/createQuiz
+// Method : POST
+export async function POST({ request, cookies }) {
+	try {
+		const payload = await request.json();
+		const { json, title, description } = payload;
+		const result = await FormationService.createQuiz(cookies, json, title, description);
+        console.log(result);
+        const idMatch = typeof result === 'string' ? result.match(/Quiz created successfully with id (\d+)/) : null;
+		if (idMatch && idMatch[1]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)